pyfixest/estimation/literals.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Literal, get_args
+
+prediction_type = Literal["response", "link"]
```

Comment:
I've just checked naming conventions of type variables and PEP suggests that 

> Names of type variables introduced in [PEP 484](https://peps.python.org/pep-0484/) should normally use CapWords (https://peps.python.org/pep-0008/#type-variable-names)

Shall we adopt this and rename the literal types accordingly (e.g., `prediction_type` to `PredictionType`)?

Reply:
I guess generally it is good to follow style suggestion (though my general opinion is that it's fine to deviate now and then). I guess it won't hurt to follow them here?